conjure-go-contract/errors/error_code.go

```diff
@@ -142,6 +148,8 @@ func (ec *ErrorCode) UnmarshalText(data []byte) error {
 		*ec = Conflict
 	case "REQUEST_ENTITY_TOO_LARGE":
 		*ec = RequestEntityTooLarge
+	case "TOO_MANY_REQUESTS":
```

Comment:
I'd like to get this added to the spec before adding to this library: https://github.com/palantir/conjure/blob/master/docs/spec/conjure_definitions.md#errorcode